integration_tests/build_test_containers.sh

```diff
@@ -2,7 +2,8 @@
 
 if [[ "$(docker images -q d3a-tests 2> /dev/null)" == "" ]]; then
   cd tests/ && \
-  git clone https://github.com/gridsingularity/d3a.git && \
+  cp -rf ~/dev/d3a . && \
+#  git clone https://github.com/gridsingularity/d3a.git && \
```

Comment:
I guess this needs to be reverted before merging?

Reply:
Reverting, thanks